src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/aggregation/AggregationSingle.java

```diff
@@ -0,0 +1,51 @@
+package com.alipay.oceanbase.rpc.protocol.payload.impl.execute.aggregation;
+
+import com.alipay.oceanbase.rpc.protocol.payload.impl.execute.aggregation.AggregationType;
+import com.alipay.oceanbase.rpc.util.Serialization;
+
+import static com.alipay.oceanbase.rpc.util.Serialization.encodeObUniVersionHeader;
+import static com.alipay.oceanbase.rpc.util.Serialization.getObUniVersionHeaderLength;
+
+public class AggregationSingle {
+    public AggregationSingle(AggregationType aggregationType_, String column_) {
```

Comment:
Shall we move all the members to the top of the class? To keep a consistent style.

Reply:
fixed